src/NHSD.BuyingCatalogue.Ordering.Domain/Order.cs

```diff
@@ -96,6 +96,13 @@ public decimal CalculateCostPerYear(CostType costType)
             return _orderItems.Where(x => x.CostType == costType).Sum(y => y.CalculateTotalCostPerYear());
         }
 
+        public decimal CalculateTotalOwnershipCost()
```

Comment:
Would it be possible to add some unit tests to cover this specific function in the domain OrderTests?